xbmc/pvr/channels/PVRChannelGroups.h

```diff
@@ -207,6 +201,18 @@ namespace PVR
     bool GetGroupsFromClients(void);
     void SortGroups(void);
 
+    /*!
+     * @brief Remove the given channels from all non-system groups.
+     * @param channel The channels to remove.
+     */
+    void RemoveFromAllGroups(const std::vector<std::shared_ptr<CPVRChannel>>& channelsToRemove);
+
+    /*!
+     * @brief Remove a channel from all non-system groups.
+     * @param channel The channel to remove.
+     */
+    void RemoveFromAllGroups(const CPVRChannelPtr& channel);
```

Comment:
Your changes looks good, only a question about `CPVRChannelPtr` and `std::shared_ptr<CPVRChannel>`, on some placed you have changed them to the `std::` and like here you use  `CPVRChannelPtr` again, When changing this accidentally went down with switch?

Reply:
Formerly I preferred the typedef (xyzPtr), if I write new code today (or change a line containing such a type), I use/change this to use std::xyz. 

I will change this line accordingly. Thanks.